reana_client/cli/workflow.py

```diff
@@ -635,8 +656,16 @@ def workflow_stop(ctx, workflow, force_stop, access_token):
 @add_access_token_options
 @click.pass_context
 def workflow_run(ctx, file, filenames, name, skip_validation,
-                 access_token, parameters, options):
-    """Create, upload and start wrapper command."""
+                 access_token, parameters, options):  # noqa: D301
+    """Create, upload and start a new analysis workflow.
+
+    The `run` command permits to create a new analysis workflow, upload its
+    input files and start it in one command.
```

Comment:
Add a phrase:

It is a shortcut for `create`, `upload` and `start` chain of commands.